site/hi.njk

```diff
@@ -0,0 +1,123 @@
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">COVID-19 महामारी को रोकने के लिए एक आंदोलन</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+
+            हमारी सरकारें केवल धीरे-धीरे <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2"> SARS-CoV-2 </a> के प्रसार और <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019"> COVID-19 महामारी </a> को नियंत्रित करने के उपायों को लागू कर रही हैं । धीमी प्रतिक्रियाओं, सार्वजनिक तुष्टिकरण की नीतियों और अर्थव्यवस्था को स्थिर करने के उनके आग्रह ने उन्हें इस बीमारी से बचाने के लिए आवश्यक उपायों को करने से रोक दिया है। हालाँकि, यह न केवल सरकार का बोझ है। इस धरती के नागरिकों के रूप में, अब कार्रवाई करने और COVID -19 से लड़ने में अपनी ओर से काम करने का समय है।
+
+                <br /><br />
+                चलो इसे कुंद करें: <strong>भाड़ में जाओ घर!</strong>
+            </p>
+
+            <h2 class="text-3xl">सेल्फ क्वारंटाइन मेनिफेस्टो</h2>
+
+            <p>
+                अच्छी तरह से अध्ययन किए गए उपचार और कम से कम एक वर्ष के लिए कोई व्यवहार्य वैक्सीन उपलब्ध नहीं होने के कारण, कोरोनोवायरस महामारी को खाड़ी में रखने का एकमात्र प्रभावी तरीका वायरस को फैलने की कम संभावना देना है। निम्नलिखित कार्यों की सूची, जो महामारी के खिलाफ लड़ाई में सबसे प्रभावी को लागू करने के लिए सबसे आसान से आदेश दिया गया है, उन लोगों के लिए ढीले दिशानिर्देशों के एक सेट के रूप में काम करना चाहिए जो आंदोलन में शामिल होना चाहते हैं और कार्रवाई कर सकते हैं जो वास्तव में एक अंतर बना सकते हैं।
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">घबराओ मत, लेकिन सतर्क रहो।</li>
+                <li class="pb-2">अपने हाथों को अक्सर धोएं और अच्छी खांसी और छींकने का अभ्यास करें।</li>
+                <li class="pb-2">अपने मुँह, नाक और आँखों सहित जितना संभव हो सके अपने चेहरे को छूने की कोशिश करें।</li>
+                <li class="pb-2">सामाजिक दूरी, कोई गले और चुंबन, कोई हाथ मिलाने, कोई उच्च फाइव्स का अभ्यास करें। यदि आपको आवश्यक हो, तो सुरक्षित विकल्पों का उपयोग करें।</li>
+                <li class="pb-2">कॉन्सर्ट, स्टेज प्ले, खेल स्पर्धा या अन्य किसी बड़े मनोरंजन कार्यक्रम में शामिल न हों।</li>
+                <li class="pb-2">संग्रहालयों, प्रदर्शनियों, मूवी थियेटर, नाइट क्लब और अन्य मनोरंजन स्थलों पर जाने से बचना चाहिए।</li>
+                <li class="pb-2">क्लब की बैठकों, धार्मिक सेवाओं और निजी पार्टियों जैसे सामाजिक समारोहों और कार्यक्रमों से दूर रहें।</li>
+                <li class="pb-2">यात्रा को कम से कम करें। यदि आवश्यक न हो तो लंबी दूरी की यात्रा न करें।</li>
+                <li class="pb-2">यदि आवश्यक न हो तो सार्वजनिक परिवहन का उपयोग न करें।</li>
+                <li class="pb-2">यदि आप घर से काम कर सकते हैं, तो घर से काम कर सकते हैं। अपने नियोक्ता से आग्रह करें कि यदि आवश्यक हो तो दूरस्थ कार्य की अनुमति दें।</li>
```

Comment:
`                <li class="pb-2">यदि आप घर से काम कर सकते हैं, तो घर से काम करें। अपने नियोक्ता से आग्रह करें कि यदि आवश्यक हो तो दूरस्थ कार्य की अनुमति दें।</li>
`

Edited.

Reply:
Changed. Thank you.